vnext/Scripts/Microsoft.ReactNative.SignConfig.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SignConfigXML>
+  <job platform="" configuration="" dest="__INPATHROOT__" jobname="microsoftreactnative" approvers="">
+    <!-- Overwrite the input files to simplify downstream things like nuget package generation. -->
+    <file src="__INPATHROOT__\ARM\Release\Microsoft.ReactNative\Microsoft.ReactNative.dll" signType="AuthenticodeFormer" dest="__INPATHROOT__\ARM\Release\Microsoft.ReactNative\Microsoft.ReactNative.dll" />
```

Comment:
Can we later add the managed dll's here as well?

Reply:
Are we planning on distributing them in the nuget? We aren't at present.

Reply:
I mean, we can sign everything regardless, (though IIRC C# dlls need different certs, I'd have to check).

Reply:
I just added the nuget package support for C# to ensure we don't have to ship any source for managed nor native.